modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/diagnostic/pagelocktracker/dumpprocessors/ToStringDumpHelper.java

```diff
@@ -24,7 +24,6 @@
  * Helper for creating string from {@link PageLockDump}.
  */
 public class ToStringDumpHelper {
-
     /** Date format. */
     public static final SimpleDateFormat DATE_FMT = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SSS");
```

Comment:
I know it's out of scope, but can you please also check usages of this formatter?

Reply:
I've fixed this particular usage, but let me change other usages in a separate PR

Reply:
Sure